go.mod

```diff
@@ -1,31 +1,32 @@
 module github.com/red-hat-storage/odf-operator
 
-go 1.22.0
+go 1.22.7
 
 require (
 	github.com/IBM/ibm-storage-odf-operator v1.6.0
 	github.com/ghodss/yaml v1.0.1-0.20220118164431-d8423dcdf344
 	github.com/go-logr/logr v1.4.2
-	github.com/onsi/ginkgo/v2 v2.20.1
-	github.com/onsi/gomega v1.34.2
-	github.com/openshift/api v0.0.0-20240926211938-f89ab92f1597
+	github.com/onsi/ginkgo/v2 v2.21.0
+	github.com/onsi/gomega v1.35.1
+	github.com/openshift/api v0.0.0-20241120064718-caf97963ed30
 	github.com/openshift/custom-resource-status v1.1.3-0.20220503160415-f2fdb4999d87
-	github.com/operator-framework/api v0.26.0
-	github.com/prometheus/client_golang v1.20.2
-	github.com/red-hat-storage/ocs-operator/api/v4 v4.0.0-20240829184418-0165a664f12c
+	github.com/operator-framework/api v0.27.0
+	github.com/prometheus/client_golang v1.20.5
+	github.com/red-hat-storage/ocs-operator/api/v4 v4.0.0-20241120130653-d486fab1e15c
 	github.com/stretchr/testify v1.9.0
 	go.uber.org/multierr v1.11.0
-	k8s.io/api v0.31.0
-	k8s.io/apiextensions-apiserver v0.31.0
-	k8s.io/apimachinery v0.31.0
-	k8s.io/client-go v0.31.0
+	k8s.io/api v0.31.2
+	k8s.io/apiextensions-apiserver v0.31.1
+	k8s.io/apimachinery v0.31.2
+	k8s.io/client-go v0.31.2
 	sigs.k8s.io/controller-runtime v0.19.0
 )
 
+require github.com/cenkalti/backoff/v4 v4.3.0 // indirect
```

Comment:
Can you add this require in the below section.

Reply:
Yeah seems like it got added automatically, also looking into the go.sum failure.